run_celery_eventlet.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# custom run script for starting corgi celery service in corgi-stage and corgi-prod environments.
+
+# Remove any left-over PID files in case the container is being restarted to prevent errors such as:
+# ERROR: Pidfile (/tmp/eventlet.pid) already exists.
+rm -f /tmp/eventlet.pid
+
+exec celery -A config worker -E --loglevel info --pidfile /tmp/eventlet.pid -Q eventlet -P eventlet -c 16 -n celery-eventlet@%h
```

Comment:
I think if we started processing slow queue messages these pods would require more memory. But you gave me a good idea that it should probably be only the software_composition_analysis task which should be in the slow queue, that way it can process eventlet tasks on the slow workers if they are idle. Also, that should allow other 'slow' task to be handled more quickly by the eventlet workers. We can probably even scrap the 'fast' queue. I'll do some testing with that.

Reply:
I reverted to only 2 queues, fast and slow. The slow queue can also handle fast tasks.